core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -443,7 +443,7 @@ object TopicCommand extends Logging {
     print("\tPartition: " + tp.partition)
     print("\tLeader: " + (if(tp.leader.isDefined) tp.leader.get else "none"))
     print("\tReplicas: " + tp.assignedReplicas.mkString(","))
-    print("\tIsr: " + tp.isr.mkString(","))
+    print("\tIsr: " + (if(tp.isr.isEmpty) "not available" else tp.isr.mkString(",")))
```

Comment:
The case we're trying to detect is here: https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/server/MetadataCache.scala#L93. Basically when there is no current leader, the broker will return an empty ISR. I think the logic here should match that. So if there is no leader defined, then we can represent the ISR as "not available" or maybe just "N/A". Otherwise, we can print the ISR as it is. Does that make sense?